certs/README.md

```diff
@@ -0,0 +1,26 @@
+# Self-signed certificates
+
+_For development environment only_
+
+The VA uses self-signed certificates with a non-globally trusted Root Certificate
+authority. 
+
+
+Used in src/site/stages/build/drupal/api.js line 54 if using the SOCKS proxy:
+```
+async proxyFetch(url, options = {}) {
```

Comment:
This is also liable to get out of date pretty easily. I recommend instead referencing `proxyFetch` (as in the suggestion above) and removing this bit.